engine/src/main/scala/cromwell/engine/workflow/workflowstore/InMemoryWorkflowStore.scala

```diff
@@ -32,7 +32,7 @@ class InMemoryWorkflowStore extends WorkflowStore {
     * Retrieves up to n workflows which have not already been pulled into the engine and sets their pickedUp
     * flag to true
     */
-  override def fetchStartableWorkflows(n: Int, cromwellId: String, heartbeatTtl: FiniteDuration)(implicit ec: ExecutionContext): Future[List[WorkflowToStart]] = {
+  override def fetchStartableWorkflows(n: Int, cromwellId: String, heartbeatTtl: FiniteDuration, excludedGroups: Set[String])(implicit ec: ExecutionContext): Future[List[WorkflowToStart]] = {
```

Comment:
Does the "in memory workflow store" respect this new parameter?

Reply:
It currently accepts it and drops it on the ground. The in-memory store is used for testing and for the single workflow runner, and I don't think it's applicable in either place. And the [underlying `var workflowStore`](https://github.com/broadinstitute/cromwell/blob/develop/engine/src/main/scala/cromwell/engine/workflow/workflowstore/InMemoryWorkflowStore.scala#L16) does not track group in an accessible manner. 

I don't feel amazing about how the interface pattern forces the in-memory version to include it in the param list as if it actually did something, but I can't think of anything better at present.

Reply:
Sounds good to me. If you felt very civic minded you could add an explicit "throw new NotImplementedException" if it's not empty, but I'll leave that up to you